packages/shared/src/models/objection/File.ts

```diff
@@ -10,24 +10,26 @@ export default class File extends BaseModel {
   cid?: string
   verified?: boolean
   sizeBytes?: number
-  originalName?: string
-  encoding?: string
-  mimeType?: string
-  expiresAt?: Date
+  originalName?: string | null
+  encoding?: string | null
+  mimeType?: string | null
+  expiresAt?: Date | null
+  lastScannedBlock?: string
 
-  // sanitize owner address on find and insert
   static findOne(args: any) {
-    if (args.owner) args.owner = args.owner.toLowerCase()
+    if (args.owner) args.owner = args.owner.toLowerCase() // sanitize address
     return super.findOne(args)
   }
   $beforeInsert: BaseModel['$beforeInsert'] = async (queryContext) => {
     await super.$beforeInsert(queryContext)
-    if (this.owner) this.owner = this.owner.toLowerCase()
-    if (!this.expiresAt) this.expiresAt = new Date(Date.now() + 1 * DAYS) // expire in 1 day without verification
+    if (this.owner) this.owner = this.owner.toLowerCase() // sanitize address
+    // expire in 1 day without verification
+    if (!this.expiresAt) this.expiresAt = new Date(Date.now() + 1 * DAYS)
```

Comment:
Let's define this as a constant somewhere so we don't miss it

Reply:
Defined `const EXPIRATION_PERIOD` at the top.